test/integration/select_test.lua

```diff
@@ -96,6 +97,23 @@ local function insert_customers(g, customers)
     return inserted_objects
 end
 
+local function insert_goods(g, skus)
+    local inserted_objects = {}
+
+    for _, sku in ipairs(skus) do
```

Comment:
What does `skus` mean?

Reply:
Since the test space is about goods, sku is an SKU (storage keeping unit), a known abbreaviation for goods units.